pkg/service/did/ion.go

```diff
@@ -52,8 +54,24 @@ type ionHandler struct {
 var _ MethodHandler = (*ionHandler)(nil)
 
 type CreateIONDIDOptions struct {
-	ServiceEndpoints []did.Service   `json:"serviceEndpoints"`
-	PublicKeys       []ion.PublicKey `json:"publicKeys"`
+	// Services to add to the DID document that will be created.
+	ServiceEndpoints []did.Service `json:"serviceEndpoints"`
+
+	// List of public keys to add to the DID document. Each element must follow the schema described in step 3 of https://identity.foundation/sidetree/spec/#add-public-keys.
+	// Each element will be used to add public keys to the DID document in the same way in which the `add-public-keys`
+	// patch action adds keys (see https://identity.foundation/sidetree/spec/#add-public-keys).
+	//
+	// It is recommended to us `jwsPublicKeys` instead of this method, which guarantees that the caller has control
+	// of the private key associated with the public keys that's being added.
+	PublicKeys []ion.PublicKey `json:"publicKeys"`
```

Comment:
do we need this if the pk is contained in the JWS?

what if the two conflict? we're needing to check for missing / mismatched keys

Reply:
Removed.